extensions/google-shopping-ads/index.md

```diff
@@ -7,7 +7,7 @@ The Google Shopping ads extension installs and adds Google Shopping features to
 
 ## Requirements
 
-- **Magento Instance**: Google Shopping ads Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.2.X and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
+- **Magento Instance**: Google Shopping ads Channel can be installed on instances with {{site.data.var.ce}} and {{site.data.var.ee}} versions 2.2.4+ and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
```

Comment:
In the `extensions/google-shopping-ads/release-notes/index.md` file, the version supported is 2.3.x with a lowercase X. Here, it's uppercase. Pick one. (I prefer lowercase.)